CISS/transforms/CISS_Constants.py

```diff
@@ -0,0 +1,13 @@
+from CISS_Injury_Value_Map import ciss_injury_col_specific_value_maps
+from CISS_Patient_Value_Map import ciss_patient_col_specific_value_maps
+
+# Injury Constants
+injury_join_columns = ['CASEID', 'CASENO', 'VEHNO', 'OCCNO', 'INJNO']
+injury_output_columns = ['CASEID', 'CASENO', 'YEAR', 'VEHNO', 'OCCNO', 'INJNO', 'AISCODE', 'AIS', 'REGION', 'STRUTYPE', 'STRUSPEC',
+                         'INJLEVEL', 'L1', 'L2', 'LDEF']
+
+# Patient Constants
+patient_join_columns = ['CASEID', 'PSU', 'CASENO', 'VEHNO', 'OCCNO']
+patient_output_columns = ['CASEID', 'YEAR', 'OCCNO', 'VEHNO', 'AGE', 'SEX', 'HEIGHT', 'WEIGHT', 'ROLE', 'PARBELTUSE',
```

Comment:
This might be intentional, but 'PSU' seems to be missing in patient_output_columns. Could be helpful to keep this information in the cleaned data for the sake of completeness.

Reply:
That makes sense to me, I'll add PSU and CASENO